### ISSUE TITLE
Outreachy Code Project: Priyansh Singh

### ISSUE THREAD

**Issue by ps-19** [ersilia-os/ersilia#208]
Applicant: @ps-19

Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the Ersilia Model Hub, a FOSS platform offering pre-trained AI/ML models for research”.

Please tick the tasks as you complete them. To make a final application it is not required to have completed all tasks. This project requires knowledge of the Python programming language. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable.

---
### Initial steps
- [ ] Record your application for the project in the Outreachy website referencing this issue. Please make sure to select the right project on the website.
- [ ] Join the Slack channel to follow public communications.
- [ ] Comment under this issue explaining why you are interested in this project.
---
### Installation of the Ersilia Model Hub
- [ ] Install the `ersilia` library.
- [ ] Add a screenshot under this issue showing that you are able to run one model (for example, the chemprop-antibiotic model)
- [ ] Fetch at least 3 models from the Ersilia Model Hub. You can find these models with the `ersilia catalog` command. Add a screenshot of the *local* catalog (`ersilia catalog –local`)
---
### CLI
- [ ] Check if there are open issues related to the command line interface. Continue with the next tasks if they are open.
- [ ] Select one issue related to improving the CLI and request to be assigned to it.
- [ ] Link the #PR as a comment under this issue.
- [ ] Make any changes required in the PR and tick this box once it has been approved.
- [ ] Suggest at least one missing feature in the CLI (one sentence is enough, for example: “Add command to estimate memory usage of a particular model”).
---
### Python library
- [ ] Add a screenshot showing that you are able to run predictions using `ersilia` as a Python library (find more information  [here](https://ersilia.gitbook.io/ersilia-book/quick-start/antibiotic-activity-prediction)). Ideally, use a Jupyter notebook.
- [ ] Create a simple [Streamlit](https://streamlit.io/) app using the `ersilia` Python library. The app can have an input and an output box, and perhaps a few models to select. Add a screenshot of the app as seen in your browser.
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Scientific content
- [ ]  Check the models available in the Hub
- [ ] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [ ] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [ ] Add a comment in this issue with your new idea, tagging the mentor 
- [ ] Get feedback from the mentor and act accordingly
- [ ] Link in the comments any other PR you have contributed to.
---
### Community
- [ ] Look up two other projects and comment on their issues with feedback on one of their tasks
- [ ] If you have feedback from your peers, answer it in this issue.
---
### Final application
- [ ] I have answered all comments from mentors and contributors
- [ ] All PR or issues assigned to me are complete
- [ ] I have submitted my final application to the project

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 1 53 30 AM](https://user-images.githubusercontent.com/63330165/162825429-cbd540ee-0612-425d-8c5d-386e314f6788.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 1 51 58 AM](https://user-images.githubusercontent.com/63330165/162825454-a1546d4f-79b5-4c1f-9f3c-d963c55b4d25.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 1 02 40 AM](https://user-images.githubusercontent.com/63330165/162825459-b6a53317-1a76-48a3-8d32-e44490334894.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 12 58 21 AM](https://user-images.githubusercontent.com/63330165/162825461-397a4b12-3948-4769-9529-da7ddf2afdc8.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 12 55 24 AM](https://user-images.githubusercontent.com/63330165/162825463-6568de65-f36e-4522-b1bc-989c75633f0d.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 12 51 38 AM](https://user-images.githubusercontent.com/63330165/162825465-e76383da-5ef1-4c95-b192-e60ed19e7929.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 12 48 38 AM](https://user-images.githubusercontent.com/63330165/162825468-64404ac0-c893-4dd4-b48e-b09c438ce95a.jpeg)

**Comment by ps-19** [comment]
![WhatsApp Image 2022-04-08 at 12 43 56 AM](https://user-images.githubusercontent.com/63330165/162825470-8c4fbc91-6d6d-40bb-b1e5-cc9482377af3.jpeg)

**Comment by ps-19** [comment]
![model_mine](https://user-images.githubusercontent.com/63330165/162826193-c217c44f-d391-4318-a668-c178dc0d8b04.png)

**Comment by ps-19** [comment]
Canva Design:
https://www.canva.com/design/DAE9NOj_TeE/0Wm5GEWoy3U0IjKH8XSl7w/edit?utm_content=DAE9NOj_TeE&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

**Comment by ps-19** [comment]
Youtube Ersilia:
https://youtu.be/wxELKx3wIOg

**Comment by ps-19** [comment]
https://medium.com/@priyansh.singh100/ai-ml-for-biomedical-research-26bb4524e443

**Comment by ps-19** [comment]
https://medium.com/@priyansh.singh100/artificial-intelligence-how-is-it-changing-medical-sciences-and-the-future-dfda48ca56f5

**Comment by ps-19** [comment]
My Merged PRs:

- https://github.com/ersilia-os/ersilia/pull/22
- https://github.com/ersilia-os/ersilia/pull/48
- https://github.com/ersilia-os/ersilia/pull/71
- https://github.com/ersilia-os/ersilia/pull/77
- https://github.com/ersilia-os/ersilia/pull/82

**Comment by ps-19** [comment]
Languages: Python C C++

Projects on Python: 
[AI/ML project](https://github.com/ps-19/Sudoku-Deep_Learning_Model)
[Django](https://github.com/ps-19/CRUD)

**Comment by GemmaTuron** [comment]
Hi @ps-19 

Thanks for the work! 
Can I ask you to not upload the video on youtube? We need to make sure we give a consisten image of Ersilia on Social Media, and if we upload all videos we are working on in the website it will confuse people who is looking for information. Simply keep it somewhere where we can access it but do not add it to the social media, many thanks!

My major feedback is regarding the use of images, have you created those graphs or are you using them from somewhere? in this case, please cite the source of information. Same for the video graphs.

I think you are ready to prepare the final submission after you amend the above comments, many thanks for your contribution!

**Comment by ps-19** [comment]
Feature Requested in CLI: https://github.com/ersilia-os/ersilia/issues/234

**Comment by ps-19** [comment]
Finally completely all of the task.
![WhatsApp Image 2022-04-21 at 11 41 02 PM](https://user-images.githubusercontent.com/63330165/164535223-c8493108-e12c-4c97-ba0b-4dd947ed20bb.jpeg)
![WhatsApp Image 2022-04-21 at 11 54 45 PM](https://user-images.githubusercontent.com/63330165/164535237-396069cf-8ed6-4df5-909c-8af289e1cdc9.jpeg)
![WhatsApp Image 2022-04-22 at 12 00 02 AM](https://user-images.githubusercontent.com/63330165/164535243-0b76ced6-276d-49ec-a64f-e749ce629d03.jpeg)
![WhatsApp Image 2022-04-22 at 12 00 47 AM](https://user-images.githubusercontent.com/63330165/164535254-b319b7e9-1ead-4eaf-8d14-d2273632deb4.jpeg)
![WhatsApp Image 2022-04-22 at 12 10 41 AM](https://user-images.githubusercontent.com/63330165/164535259-cf2d69c7-0948-4b5f-98c0-0ad237b52736.jpeg)
![WhatsApp Image 2022-04-22 at 12 13 29 AM](https://user-images.githubusercontent.com/63330165/164535264-0753bfea-1d96-4583-8798-58811f952133.jpeg)
![WhatsApp Image 2022-04-22 at 12 15 07 AM](https://user-images.githubusercontent.com/63330165/164535269-6e46ecf9-8ce7-4b5e-87cc-f9cba374f408.jpeg)

**Comment by ps-19** [comment]
Feature Requested: https://github.com/ersilia-os/ersilia/issues/254

**Comment by ps-19** [comment]
https://docs.google.com/document/d/1u0UfmoJg512SbeSeDnKxJNI1OSVUk8jwUZG_pYwh4Z0/edit?usp=sharing

**Comment by ps-19** [comment]
I am an undergraduate student of IT. In my first year I started doing competitive coding and still today I am doing competitive coding.

In my second  year I got to know about open source from GSoC and outreachy internship,
I started contributing to repositories like kubernetes-client, Pycharm, Cherrypy, LoginRadius, Cataclysm-DDA etc. I have a lot of PRs merged into these repositories(https://github.com/ps-19).

In my third year I am doing software development using Nodejs, Express, React, MongoDB, Django etc. and made some projects on them
and right now I am working on some projects on Deep Learning (github.com/ps-19).
I am doing competitive coding and contributing to open source as well,
I have experience in complete(backend and frontend) software development and web development using MERN stack and other techs.

I'm passionate about the application of ML in healthcare, drug discovery, genetic engineering etc . I wish to further pursue post grad in one of the fields like computational biology, biotechnology, AI+health(not sure, still exploring) but somewhere along the lines. This project gives me an opportunity to work closely with the ML papers in drug discovery.
I'm fluent in python and very well familiar with implementing ML papers, ML Toolboxes. This project is apt for my skill set and interest.

My github account is https://github.com/ps-19 and my resume could be found here https://bit.ly/3IiJ6XD, email is (priyansh.singh100@gmail.com)
and portfolio here https://ps-19.github.io/Portfolio/. Kindly give me at least one opportunity to prove myself.
